src/NUnitFramework/framework/Internal/TestProgressReporter.cs

```diff
@@ -39,19 +43,39 @@ public void TestStarted(ITest test)
             var parent = GetParent(test);
             try
             {
-                string report;
+                var report = _testStartedStringBuilder ??= new StringBuilder();
```

Comment:
question: Should we set an initial capacity? I think it'll default to `16` but in either path below we likely require more

Reply:
With MS version it would be 16, without initial the call after first will have larger StringBuilder with this original code.